hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -209,7 +208,7 @@ public static String randomNameOwnedBy(HazelcastInstance instance, String prefix
         for (; ; ) {
             String id = prefix + randomString();
             Partition partition = partitionService.getPartition(id);
-            if (comparePartitionOwnership(false, localMember, partition)) {
+            if (comparePartitionOwnership(true, localMember, partition)) {
```

Comment:
Nobody else is using this method since I introduced it yesterday. I can now retrieve the name of the executing test:

```
 @Before
    public void setup() {
        Config config = new Config();
        config.addListConfig(new ListConfig("testAdd_whenCapacityReached_thenItemNotAdded*").setMaxSize(10));

        instances = newInstances(config);
        HazelcastInstance local = instances[0];
        HazelcastInstance target = instances[instances.length - 1];
        String methodName = getThreadLocalFrameworkMethod().getMethod().getName();
        String name = randomNameOwnedBy(target, methodName);
        list = local.getList(name);
    }
```

So depending on the test you are executing, you get the right configured element.